src/mbgl/gl/uniform_buffer_gl.cpp

```diff
@@ -23,5 +24,12 @@ UniformBufferGL::~UniformBufferGL() {
     }
 }
 
+void UniformBufferGL::update(const void* data, std::size_t size_) {
+    assert(size == size_);
```

Comment:
Is `size_>size` a memory corruption and potential crash?  Maybe we should return or limit the size in release builds.

Reply:
I agree: assert + log error? + return